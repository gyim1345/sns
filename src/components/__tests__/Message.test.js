import React from 'react';
import { mount } from 'enzyme';
import Message from '../Message';

describe('<Message />', () => {
  it('스냅샷 비교', () => {
    const wrapper = mount(<Message msg={'Message'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
