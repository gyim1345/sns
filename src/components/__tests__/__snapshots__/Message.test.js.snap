// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> 스냅샷 비교 1`] = `
<Message
  msg="Message"
>
  <div
    className="alert alert-info alert-dismissible fade show"
    role="alert"
  >
    Message
    <button
      aria-label="Close"
      className="close"
      data-dismiss="alert"
      type="button"
    >
      <span
        aria-hidden="true"
      >
        ×
      </span>
    </button>
  </div>
</Message>
`;
